src/Components/ServiceSelectionScene/ServiceSelectionScene.tsx

```diff
@@ -246,10 +247,15 @@ export class ServiceSelectionComponent extends SceneObjectBase<ServiceSelectionC
   // Creates a layout with logs panel
   buildServiceLogsLayout(service: string) {
     return new SceneCSSGridItem({
+      $behaviors: [new behaviors.CursorSync({ sync: DashboardCursorSync.Off })],
```

Comment:
@svennergr Disabled it using this. Took inspiration from here: https://github.com/grafana/explore-logs/blob/main/src/Components/ServiceScene/ServiceScene.tsx#L447